tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/hive/HiveTableManager.java

```diff
@@ -114,17 +107,11 @@ public HiveTableInstance createMutable(HiveTableDefinition tableDefinition, Stat
             }
         }
         else if (state == LOADED) {
+            String tableDataPath = getMutableTableHdfsPath(tableName, Optional.empty());
```

Comment:
you could assert that the data path doesn't exists (if there isn't such check already)